src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationService.java

```diff
@@ -28,8 +37,17 @@ public MigrationService(WorkflowClient workflowClient,
 
     public void createMigrationCase(MigrationData migrationCaseData, MigrationCaseTypeData migrationCaseTypeData) {
 
-        log.info("create Migration Case, started : type {}", migrationCaseData.getComplaintType());
+        // dummy documents list
+        DocumentSummary documentSummary = new DocumentSummary("migration","","");
+        var migrationRequest = composeMigrateCaseRequest(migrationCaseData, migrationCaseTypeData, documentSummary);
+        CreateCaseworkCaseResponse caseResponse = caseworkClient.migrateCase(migrationRequest);
+        log.info("Created migration case {}", caseResponse.getUuid());
```

Comment:
I'm not overly convinced of the value of this - we would be able to follow the logs to find this out.

Reply:
We can remove this in the future if it becomes overbearing in the logs.